src/near_indexer/src/helper/proof.rs

```diff
@@ -3,13 +3,11 @@ use std::env;
 use super::{TweetResponse, ZkInputParam};
 use super::methods::VERIFY_ELF;
 use alloy_sol_types::SolValue;
-use k256::SecretKey;
-use rand::rngs::OsRng;
 use risc0_ethereum_contracts::groth16;
 use risc0_zkvm::{default_prover, ExecutorEnv, ProverOpts, VerifierContext};
 use std::error::Error;
 use tracing::debug;
-use verity_client::client::{AnalysisConfig, VerityClient, VerityClientConfig};
+use verity_client::client::{ VerityClient, VerityClientConfig};
```

Comment:
Use `rust-analyzer` for auto-formatting across project.

Reply:
The formatting is active onSave sometimes it just failed. 
I could set up a format git hook and all our projects to prevent this permanently